les/server.go

```diff
@@ -85,6 +85,10 @@ type LesServer struct {
 }
 
 func NewLesServer(node *node.Node, e ethBackend, config *ethconfig.Config) (*LesServer, error) {
+	lespayDb, err := node.OpenDatabase("lespay", 0, 0, "eth/db/lespay.server")
```

Comment:
Perhaps would be nice to use `node.OpenDatabase("lesdb", 0, 0, "eth/db/les.server")`. And also update in the client side?